base/kra/src/main/java/org/dogtagpki/server/kra/rest/KeyService.java

```diff
@@ -629,15 +630,29 @@ public KeyInfo createKeyDataInfo(KeyRecord rec, boolean getPublicKey) throws EBa
         return ret;
     }
 
-    private String createSearchFilter(String status, String clientKeyID, String realm) {
+    private String createSearchFilter(String status, String clientKeyID, String realm, String ownerName) {
         String filter = "";
         int matches = 0;
+        Object tokenID = "10000000000000000002";
+        Object userID = "foouser1";
 
         if ((status == null) && (clientKeyID == null)) {
             filter = "(serialno=*)";
             matches ++;
         }
 
+        if (ownerName == null) {
+            filter = "(keyownerName=*)";
+            matches ++;
+            return filter;
+        }
```

Comment:
I think this is not needed. Instead, the code above it should be modified like this:
```
if ((status == null) && (clientKeyID == null) && (ownerName == null)) {
    filter = "(serialno=*)";
    matches ++;
}
```
This is needed to match this VLV by default (if there's no params specified):
https://github.com/dogtagpki/pki/blob/master/base/kra/database/ds/vlv.ldif#L7

Reply:
Indeed. Fixed the code. Thanks.